lib/puppetserver/ca/errors.rb

```diff
@@ -0,0 +1,11 @@
+module Puppetserver
+  module Ca
+    class Error < StandardError; end
+    class FileNotFound < Error; end
+    class InvalidX509Object < Error; end
+    class ConnectionFailed < Error; end
+
+    module Errors
```

Comment:
what were you doing here, Justin?

Reply:
Oh, I think I was going to move [CliParsing.handle_errors](https://github.com/puppetlabs/puppetserver-ca-cli/blob/master/lib/puppetserver/ca/utils/cli_parsing.rb#L53) here. I'll do that unless anyone objects.